pynautobot/core/response.py

```diff
@@ -361,6 +362,30 @@ def fmt_dict(k, v):
         init = Hashabledict({fmt_dict(k, v) for k, v in self.serialize(init=True).items()})
         return set([i[0] for i in set(current.items()) ^ set(init.items())])
 
+    def updates(self):
```

Comment:
I know your `Endpoint.update()` tests will technically invoke this method, but we should still have a unit test for this method directly as well.